wp-content/themes/redmond/views/layout/footer.html.twig

```diff
@@ -1,3 +1,39 @@
 <footer class="footer">
-
-</footer>
\ No newline at end of file
+    <div class="footer-icons">
+        <div class="row">
+            {% for column in SP.ACF.get_field('footer_icons', 'option') %}
+                <div class="medium-4 columns icon-title">
+                    <a href="#"><img class="footer-icon" src="{{ column.icon.url }}" /></a>
+                    <h2><small>{{ column.small_header }}</small> {{ column.large_header }}</h2>
+                </div>
+            {% endfor %}
+        </div>
+    </div>
+    <div class="row footer-content">
+        {% for column in SP.ACF.get_field('footer_list', 'option') %}
+            <div class="column {% if loop.index == 1 %}hide-for-small-only medium-3 medium-push-3{% endif %}{% if loop.index == 2 %}hide-for-small-only medium-3 medium-push-3{% endif %}{% if loop.index == 3 %}small-6 medium-3 medium-push-3{% endif %}">
```

Comment:
instead of doing three if checks use elseif statements. this line is also too long. put the if and elseif statements on a new line and use indenting. Have the closing `>` at the same level as the opening tag.